org/polypheny/prism/value.proto

```diff
@@ -288,3 +279,47 @@ message ProtoFile {
   // The content of the file in binary.
   bytes binary = 1;
 }
+
+/*
+ * This message represents a node in the labeled property graph model.
+ * Apart form a unique identifier and a name, a set of entries and list are used to represent properties and labels.
+ */
+message ProtoNode {
+  // The system internal unique identifier for that node.
+  string id = 1;
+  // The name of the node as specified on creation.
+  string name = 2;
+  // A map of key-value pairs of proto values. These are used as a map to store properties with their corresponding names.
+  map<string, ProtoValue> properties = 3;
+  // This list contains all labels associated with this node. Normal strings are used instead of ProtoStrings.
+  repeated string labels = 4;
+}
+
+/*
+ * This message represents an edge in the labeled property graph model.
+ * Apart form a unique identifier and a name, a set of entries and list are used to represent properties and labels.
+ * Source, target and direction fields provide information about the nodes connected by this edge.
+ */
+message ProtoEdge {
+  // The system internal unique identifier for that node.
+  string id = 1;
+  // The name of the node as specified on creation.
+  string name = 2;
+  // A map of key-value pairs of proto values. These are used as a map to store properties with their corresponding names.
+  map<string, ProtoValue> properties = 3;
+  // This list contains all labels associated with this node. Normal strings are used instead of ProtoStrings.
+  repeated string labels = 4;
+  // The system internal unique identifier of the node from which this edge originates.
+  string source = 5;
+  // The system internal unique identifier of the node to which this edge is incident.
+  string target = 6;
```

Comment:
They should be called `left` and  `right` instead of `source` and target` to match the `Direction` enum.